packages/components/psammead-media-player/src/Amp/index.jsx

```diff
@@ -23,7 +23,12 @@ const AmpMediaPlayer = ({ src, placeholderSrc }) => {
         src={src}
         allowfullscreen="allowfullscreen"
       >
-        <amp-img layout="fill" src={placeholderSrc} placeholder />
+        <amp-img
+          layout="fill"
+          src={placeholderSrc}
+          srcset={placeholderSrcset}
```

Comment:
We should probably cover this additional attribute in the AMP unit tests too.

Reply:
done.